### ISSUE TITLE
Make MonacoEditor scroll with parent container when mouse hovers&scrolls

### ISSUE THREAD

**Issue by JVimes** [go-gitea/gitea#28479]
### Description

When editing a file with the mouse cursor over the editor, the mouse wheel only affects the editor and not the web page. I'd expect it to affect the web page once the editor is scrolled to its top/bottom.

1. Click pencil icon on a text/markdown file
2. Place the mouse cursor over the document
3. Try to scroll to the bottom of the webpage using the mouse wheel

Actual: It only scrolls the document pane.
Expected: Once document is scrolled to its bottom, the web page scrolls.

### Gitea Version

1.21.1

### Can you reproduce the bug on the Gitea demo site?

Yes

### How are you running Gitea?

It happens on https://try.gitea.io/

### Database

None

**Comment by wxiaoguang** [comment]
That's how MonacoEditor works, you can try it here: https://microsoft.github.io/monaco-editor/ , unless you can figure a config option to control the scrolling behavior.

**Comment by JVimes** [comment]
[Looks like](https://github.com/microsoft/monaco-editor/issues/1853#issuecomment-593484147) the flag is `editor.scrollbar.alwaysConsumeMouseWheel: false`.

**Comment by silverwind** [comment]
Makes sense to "bubble" this scroll, especially because we already set `scrollBeyondLastLine: false`.
